### ISSUE TITLE
Negative one-body integrals!

### ISSUE THREAD

**Issue by TariniHardikar** [ProjectQ-Framework/FermiLib-Plugin-Psi4#15]
Hi,

I'm getting negative one body integrals and negative overlap matrices when I run a simple calculation on hydrogen. I created a molecule by supplying geometry, basis (sto-3g), charge and multiplicity. I ran a SCF calculation (which converged with a reasonable energy). However, when I load this molecule and print out the overlap matrix and one-body tensor, they are diagonal matrices with all negative values on the diagonal. The two-body tensor seems to be okay.

Why is this happening? And if the stored overlap matrix had negative eigenvalues, how did the SCF calculation converge nicely?

Thank you!

**Comment by jarrodmcc** [comment]
Hi,

Thank you for your feedback and help in improving the code.  It's actually okay that the one-body integrals are negative, but I looked into the issue of the overlap matrices and it turned out that was an error in FermiLib that was accidentally loading the one_body_integrals in place of the overlap integrals, because the overlap integrals were never loaded.  This is because we made a design decision some time ago to only work with orthonormal basis sets (specifically we load the orbitals in the hartree-fock canonical basis), which means the overlap matrix is always the identity.  To reduce confusion and eliminate this problem, we will remove the overlap_integrals field altogether from molecular data.  Thanks again!
